apps/app/components/states/create-update-state-inline.tsx

```diff
@@ -228,22 +229,27 @@ export const CreateUpdateStateInline: React.FC<Props> = ({
           name="group"
           control={control}
           render={({ field: { value, onChange } }) => (
-            <CustomSelect
-              value={value}
-              onChange={onChange}
-              label={
-                Object.keys(GROUP_CHOICES).find((k) => k === value.toString())
-                  ? GROUP_CHOICES[value.toString() as keyof typeof GROUP_CHOICES]
-                  : "Select group"
-              }
-              input
-            >
-              {Object.keys(GROUP_CHOICES).map((key) => (
-                <CustomSelect.Option key={key} value={key}>
-                  {GROUP_CHOICES[key as keyof typeof GROUP_CHOICES]}
-                </CustomSelect.Option>
-              ))}
-            </CustomSelect>
+            <Tooltip tooltipContent={groupLength === 1 ? "Cannot have an empty group." : "Choose State"}  >
+              <div>
+                <CustomSelect
+                  disabled={groupLength === 1}
+                  value={value}
+                  onChange={onChange}
+                  label={
```

Comment:
Please make the text muted when the select dropdown is disabled. Use the class `text-brand-secondary`.

Reply:
Ohh sure, just to confirm, I made the the changes such that the text within that dropdown should have a muted appearance...and I've also updated the PR's description! Thanks